### PR TITLE
Rust joseph

### PR DESCRIPTION
**Pull request by jastier**
Extractions are now created from Dialog Agent REST API input
These are output to the screen for now.

### CODE REVIEW

File: rust/tomcat/src/main.rs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-//! ToMCAT NLU agent
+//! ToMCAT N
```
Comment (adarshp) [inline rust/tomcat/src/main.rs:1]: Can you fix this line breaking?

File: rust/tomcat/src/main.rs [hunk 2]
```diff
@@ -67,18 +78,39 @@ fn process_chat_message(
     checker: &mut SpellChecker,
     cfg: &Config,
 ) {
+    //println!("process_chat_message start");
     // For ASIST Study 4, only the shop stage will have free text chat.
     if let MissionStage::shop_stage = mission_stage {
         let message: ChatMessage = get_message(&msg);
-        let text = message.data.text;
+//        let text = message.data.text;
```
Comment (adarshp) [inline rust/tomcat/src/main.rs:85]: This file is a bit outdated. Can you merge the changes from the latest version of the `rust` branch into `rust_joseph`?

**Review by adarshp — CHANGES_REQUESTED** [review]
Thanks @jastier  - almost there. I changed the target branch from `master` to `rust`. Can you address the comments in this review? Then we should be good to merge.

File: rust/tomcat/src/main.rs (hunk 2 above)
Comment (adarshp) [reply rust/tomcat/src/main.rs:85]: Actually, nevermind, I was wrong. The file is up to date.

File: rust/tomcat/src/config.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
```
[the remark this answers is not in the document]
Comment (jastier) [reply rust/tomcat/src/config.yml]: Done

File: rust/tomcat/src/main.rs (hunk 1 above)
Comment (jastier) [reply rust/tomcat/src/main.rs]: Done

**Review by adarshp — APPROVED** [review]

**Pull request merged**